marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/command/AbstractInstallerCommand.java

```diff
@@ -93,7 +93,8 @@ protected ObjectNode canInstallDhs(){
             installedHubVersion = versions.getInstalledVersion();
         }
         catch (Exception e){
-            logger.info("Data Hub is not installed yet");
+            logger.error("Unable to determine the version of DHF: " + e.getMessage());
+            logger.info("MarkLogic Server version will only be checked for determining compatibility");
```

Comment:
I think we need to tweak this message - should say "Will only check MarkLogic version to determine if Data Hub can be installed" (instead of saying what the only thing we're going to do with ML version is).